src/zkevm_specs/state_circuit.py

```diff
@@ -171,25 +171,22 @@ def assert_in_range(x: FQ, min_val: int, max_val: int) -> None:
 
 @is_circuit_code
 def check_start(row: Row, row_prev: Row):
-    # 1.0. field_tag is 0 for Start
+    # 1.0. Unused keys are 0
```

Comment:
Why did this comment change if we are still checking the same?

Reply:
I put all zero checks together to avoid unnecessary increasing of serial numbers.
In check start, it checks field_tag, address and id, storage_key are zero.
If we don't put together, it would increases 4 serial numbers and be confusing.
I described which are the unused keys in here and put them together as one constraint.
https://github.com/privacy-scaling-explorations/zkevm-specs/pull/385/files#diff-383bd0a8138e9b40964a4046a8b879413643629705529274f64ba6270fb350eaR46

Reply:
Ohh I see. Makes sense! Thanks for the explanation